client/components/formatted-header/index.tsx

```diff
@@ -1,26 +1,42 @@
 import clsx from 'clsx';
-import PropTypes from 'prop-types';
+import { ElementType, FC, PropsWithChildren, ReactNode } from 'react';
```

Comment:
Should these not be `type` imports?
```suggestion
import type { ElementType, FC, PropsWithChildren, ReactNode } from 'react';
```